### ISSUE TITLE
Feature Request: Allow Showing Only Trackers From Subproject While Sharing

### ISSUE THREAD

**Issue by mountainraptor** [backlogs/redmine_backlogs#827]
platform:  # supported: 2.0.4, 2.1.6, 2.2.2
backlogs:  # supported: 0.9.35
ruby:  # supported: 1.9.3, 1.8.7

I have a hierarchical project consisting of one main project and multiple sub-projects.  I use shared versions at the top-level and all sub-projects sprint together, inheriting the version from the top-level project.  When viewing backlogs at the top-level project (as the overall project manager) I can see all the subproject trackers in each sprint and in the backlogs.  This is expected behavior.  When I view backlogs from a subproject the Product Backlog displays projects for only the subproject (not the sibling subproject trackers or the master project trackers).  However, the sprints show backlogs from all the sibling projects.  I would suggest a way for the backlogs view for any subproject to show only trackers for that subproject.

TLDR; Setting to allow subproject to only show that subproject's trackers in backlogs.
